app/global/MainModule.java

```diff
@@ -460,7 +462,7 @@ public ActorRef get() {
         @Override
         public ActorRef get() {
             final Cluster cluster = Cluster.get(_system);
-            final int actorCount = _configuration.getInt("rollups.worker.count");
+            final int actorCount = _configuration.getInt("rollup.worker.count");
```

Comment:
Were all the other config references to 'rollup.' and not 'rollups.'?

Reply:
Correct.  They were all referencing "rollup"